server/util/util.js

```diff
@@ -0,0 +1,8 @@
+const jwt = require("jsonwebtoken");
```

Comment:
i would change this file route to utils/updateModel.   also since theres only one function now you can default export.

Reply:
changed it to default